implementations/rust/ockam/ockam_command/tests/bats/load/orchestrator.bash

```diff
@@ -43,7 +43,14 @@ function get_project_data() {
 
 function copy_enrolled_home_dir() {
   if [ ! -z "${ORCHESTRATOR_TESTS}" ]; then
-    cp -r $OCKAM_HOME_BASE/application_database.sqlite3 $OCKAM_HOME/
-    cp -r $OCKAM_HOME_BASE/database.sqlite3 $OCKAM_HOME/
+    cp -a $OCKAM_HOME_BASE/application_database.sqlite3 $OCKAM_HOME/
+    if [ -f $OCKAM_HOME_BASE/application_database.sqlite3-wal ]; then
```

Comment:
Hm. Is it required? I thought we only want to clone the database here

Reply:
the `sqlite3-wal` file is a piece of the database, without it the database would be corrupted.